time_test.go

```diff
@@ -0,0 +1,50 @@
+package graphql
+
+import (
+	"testing"
+	"time"
+)
+
+var timeTestCases = []interface{}{
+	time.Now(),
+	int32(1),
+	time.Now().Unix(),
+	time.Now().UnixNano(),
+	float64(-1),
+	"2006-01-02T15:04:05Z",
```

Comment:
Could you, pelase, check the expected values too? So that each test has `got`, `want` pair. This way you will not only check that marshalling and unmarshalling work but also that they return what you expect.

Reply:
Also, I wouldn't use `time.Now()` instead I prefer constants, please

Reply:
Now I added the checks for expeced values with assertion library and remove time.Now() usage.
Can you please check if it is ok for you, Thx!